crates/cargo-gpu/src/main.rs

```diff
@@ -132,35 +127,46 @@ fn cache_dir() -> std::path::PathBuf {
         .join("rust-gpu")
 }
 
+/// Location of the target spec metadata files
+fn target_spec_dir() -> std::path::PathBuf {
+    let dir = cache_dir().join("target-specs");
+    std::fs::create_dir_all(&dir).unwrap();
+    dir
+}
+
+/// Convenience function for internal use. Dumps all the CLI usage instructions. Useful for
+/// updating the README.
 fn dump_full_usage_for_readme() {
-    use clap::CommandFactory;
+    use clap::CommandFactory as _;
     let mut command = Cli::command();
 
-    let mut buffer: Vec<u8> = Default::default();
+    let mut buffer: Vec<u8> = Vec::default();
     command.build();
 
     write_help(&mut buffer, &mut command, 0);
-    let buffer = String::from_utf8(buffer).unwrap();
-    println!("{}", buffer);
+    println!("{}", String::from_utf8(buffer).unwrap());
 }
 
+/// Recursive function to print the usage instructions for each subcommand.
 fn write_help(buffer: &mut impl std::io::Write, cmd: &mut clap::Command, _depth: usize) {
     if cmd.get_name() == "help" {
         return;
     }
 
-    let mut command = cmd.get_name().to_string();
-    let _ = writeln!(
+    let mut command = cmd.get_name().to_owned();
+    writeln!(
         buffer,
         "\n* {}{}",
         command.remove(0).to_uppercase(),
         command
-    );
-    let _ = writeln!(buffer);
-    let _ = cmd.write_long_help(buffer);
+    )
+    .unwrap();
+    writeln!(buffer).unwrap();
+    cmd.write_long_help(buffer).unwrap();
 
     for sub in cmd.get_subcommands_mut() {
-        let _ = writeln!(buffer);
+        writeln!(buffer).unwrap();
+        #[expect(clippy::used_underscore_binding, reason = "Used in recursion only")]
```

Comment:
I've never seen this macro/annotation before (`expect`) - cool :)

Reply:
It's really new actually, like just became stable a few months ago.